README.md

```diff
@@ -56,7 +56,7 @@ config = {
   ]
 }
 
-rs.initiate(config)
+rs.initiate(config1)
```

Comment:
same here

Reply:
addressed in https://github.com/jembi/openhim-mediator-mapping/pull/205/commits/dd30cd98f267f8a46b99a1cb686be24a5a90613c